.github/workflows/fireci.yml

```diff
@@ -18,8 +18,10 @@ jobs:
       - uses: actions/checkout@v3.0.2
       - uses: actions/setup-python@v2
         with:
-          python-version: '3.9'
+          python-version: '3.8'
```

Comment:
so have we been running on 3.8 for release build and 3.9 for GHA workflows?

Reply:
I believe so.

We didn't specify a python version in the CI [dockerfile](https://team.git.corp.google.com/firebase-android-core/test-infra/+/refs/heads/master/images/ci/Dockerfile). From the cloud build log for the latest image ([registry](https://pantheon.corp.google.com/gcr/images/fireescape-c4819/global/ci?project=fireescape-c4819), [latest build](https://pantheon.corp.google.com/cloud-build/builds/5fd0f503-5258-4a6c-a6d5-0b20d2c1b37e?project=fireescape-c4819)), I think it was 3.8 that has been run for release.

I tried update the docker file. Building with 3.9 works ([log](https://pantheon.corp.google.com/cloud-build/builds;region=global/5bbca581-dd64-4d8d-ad6b-aaac0d272c03?project=fireescape-c4819)), but 3.10 ([log](https://pantheon.corp.google.com/cloud-build/builds;region=global/d6f9aa1c-8cec-4547-8722-1c9a08fb0d0c?project=fireescape-c4819)) and 3.11 ([log](https://pantheon.corp.google.com/cloud-build/builds;region=global/33075f22-3d1c-4157-a8f5-a2455b9b9764?project=fireescape-c4819)) seem not.